apps/condo/domains/billing/schema/resolvers/propertyResolver.js

```diff
@@ -208,8 +216,8 @@ class PropertyResolver extends Resolver {
                 if (!updated.has(existingProperty.id)) {
                     const updateInput = this.buildUpdateInput({
                         importId: importIdInput,
-                        globalId,
-                        address: resultAddressKey !== existingProperty.addressKey ? address : null,
+                        globalId: normalizePropertyGlobalId(globalId),
+                        address: resultAddressKey !== existingProperty.addressKey ? this.addressFieldValue(address, resultAddressKey) : null,
```

Comment:
We are trying to set the address to null in case the address wasn't changed. Is it ok?

Reply:
It will be compared with existing property. So, it's ok